x-pack/plugin/esql/qa/testFixtures/src/main/resources/exponential_histogram.csv-spec

```diff
@@ -0,0 +1,16 @@
+loadFiltered
+required_capability: exponential_histogram
+
+FROM exp_histo_sample | WHERE STARTS_WITH(instance, "dummy") | KEEP instance, responseTime
+;
+ignoreOrder:true
```

Comment:
Sort is not supported yet due to a dependency on #137313 . I will replace `ignoreOrder` with a sort when that PR is merged.